core/page.hpp

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017 Husky Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "base/disk_store.hpp"
+#include "base/log.hpp"
+#include "base/serialization.hpp"
+#include "core/context.hpp"
+
+namespace husky {
+
+class ObjListBase;
```

Comment:
Okay, then please add comments for this forward declaration such as where the ObjListBase is.